### DISCUSSION TITLE
How can I set a username and password requirement?

### DISCUSSION THREAD

**Discussion by rpgdev** [rustdesk/rustdesk#8226] [Q&A]
I'm using rustdesk-docker on an debian server and I'm not sure how to setup a username and password requirement. Right now I'm only using the server's url which kind of freaks me out a little bit lol. Although the clients do have pw setup, is the server not meant to be protected with credentials as well?

**Comment by xlionjuan** [comment]
Pro only feature

**Reply by rpgdev** [reply]
Lol. So sneaky. Let's start my search for alternatives to this.
